1-trackers/no-js/README.md

```diff
@@ -12,16 +12,16 @@ The wizard source code can be found [here][wizard]. The logic for generating the
 
 ## Find out more
 
-| Technical Docs              | Setup Guide           | Roadmap & Contributing               |         
-|-----------------------------|-----------------------|--------------------------------------|
-| ![i1][techdocs-image]      | ![i2][setup-image]   | ![i3][roadmap-image]                |
-| [Technical Docs][techdocs] | [Setup Guide][setup] | _coming soon_                        |
+| Technical Docs             | Setup Guide          |
+|----------------------------|----------------------|
+| ![i1][techdocs-image]      | ![i2][setup-image]   |
+| [Technical Docs][techdocs] | [Setup Guide][setup] |
 
-Also see [this introductory blog post][intro-blog-post]. A hosted version of the wizard can be found [on the SnowPlow Analytics website][hosted-wizard].
+Also see [this introductory blog post][intro-blog-post].
 
 ## Copyright and license
 
-The No-JavaScript Tracker is copyright 2012-2013 SnowPlow Analytics Ltd.
+The No-JavaScript Tracker is copyright 2012-2021 SnowPlow Analytics Ltd.
```

Comment:
I do like finding a `SnowPlow` reference. It should be `Snowplow` now :-)

Reply:
Thank you! I didn't notice! i'll add a commit now to replace them all.